packages/compass-e2e-tests/helpers/commands/set-connect-form-state.ts

```diff
@@ -1,8 +1,29 @@
+import _ from 'lodash';
 import { expect } from 'chai';
 import type { CompassBrowser } from '../compass-browser';
 import * as Selectors from '../selectors';
 import type { ConnectFormState } from '../connect-form-state';
 
+async function waitForElementAnimations(browser: CompassBrowser, element: any) {
```

Comment:
I can't figure out what the type of element should be. I thought it would be Element<'async'> to mirror the type of Browser, but then it complains (with a warning, not error) that it is WebdriverIO.Element which is unsafe. But I can't figure out how to even import that to try it.